autode/pes_2d.py

```diff
@@ -205,14 +210,11 @@ def get_ts_guess_2d(reactant, product, active_bond1, active_bond2, name, method,
         (autode.transition_states.ts_guess.TSguess)
     """
     logger.info(f'Getting TS guess from 2D relaxed potential energy scan, using active bonds '
-                f'{active_bond1} and {active_bond2}')
-
-    curr_dist1 = reactant.get_distance(atom_i=active_bond1[0], atom_j=active_bond1[1])
-    curr_dist2 = reactant.get_distance(atom_i=active_bond2[0], atom_j=active_bond2[1])
+                f'{bond1} and {bond2}')
 
-    # Steps of +Δr if the final distance is greater than the current else -Δr
-    n_steps1 = int(np.abs((final_dist1 - curr_dist1) / dr))
-    n_steps2 = int(np.abs((final_dist2 - curr_dist2) / dr))
+    # Steps of +Δr if the final distance is greater than the current else -Δr. Must be > 0
+    n_steps1 = max(int(np.abs((bond1.final_dist - bond1.curr_dist) / dr)), 1)
```

Comment:
This is just a 1D scan, if n_steps = 1?

Reply:
Yep. Better to run a pure 1D scan?

Reply:
or set n_steps as a min of 2/3?

Reply:
Good idea. Not sure how the polyfit will survive with a 1xn grid.. will add to the next PR